data/saas/config/saas_erasure_order_config.yml

```diff
@@ -0,0 +1,79 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: SaaS Erasure Order Config
+  type: custom
+  description: A sample schema for testing
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+
+  client_config:
+    protocol: https
+    host: <domain>
+
+  test_request:
+    method: GET
+    path: /test/
+
+  endpoints:
+    - name: orders
+      erase_after:
```

Comment:
New `erase_after` field that maps to `collection.erase_after` in the graph generated from the SaaS config. Should we update the existing `after` field to `access_after` for consistency?

Reply:
Do we need to prioritize this - is anyone needing the "after" functionality right now? It's something we advertise..

Reply:
I haven't heard any reports of issues with the "after" functionality. Sentry is the only connector that uses it but it looks like that traversal is enforced by the dataset references instead of the after field.

```
- name: person
  after: [<instance_fides_key>.projects]
  requests:
    read:
      method: GET
      ignore_errors: true
      path: /api/0/projects/<organization_slug>/<project_slug>/users/
      grouped_inputs: [organization_slug, project_slug, query]
      query_params:
        - name: query
          value: email:<query>
      param_values:
        - name: organization_slug
          references:
            - dataset: <instance_fides_key>
              field: projects.organization.slug
              direction: from
        - name: project_slug
          references:
            - dataset: <instance_fides_key>
              field: projects.slug
              direction: from
```

Reply:
OK I suspect no one's running into it because no one's using it yet - i'm going to see if i can get this prioritized